cypress/integration/tests/administrator/proxy/proxy.enable.disable.test.ts

```diff
@@ -32,42 +31,55 @@ describe("Proxy operations", () => {
 
     it("Http Proxy port and host field validation", function () {
         Proxy.open();
-        proxy.enableSwitch(ProxyViewSelectors.httpSwitch);
-        proxy.fillHost(ProxyType.http, proxy.hostname);
-        proxy.fillPort(ProxyType.http, "Invalid port");
+        httpProxy.enable();
```

Comment:
Looks Good @abrugaro .
One comment : I think methods like enable() and disable() should open the proxy page too .
So they will call open() inside them .

And open method will check if the page is already open don't do the navigation again .
Example : 
https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/integration/models/administrator/credentials/credentials.ts#L74